pypdf/filters.py

```diff
@@ -483,7 +483,11 @@ def decode(
             data = data.encode()
         data = data.strip(WHITESPACES_AS_BYTES)
         try:
-            return a85decode(data, adobe=True, ignorechars=WHITESPACES_AS_BYTES)
+            return a85decode(
+                            bytes(byte for byte in data if byte not in WHITESPACES_AS_BYTES),
+                            adobe=True,
+                            ignorechars=WHITESPACES_AS_BYTES,
```

Comment:
This indentation looks much too high. It is sufficient to have


```suggestion
                ignorechars=WHITESPACES_AS_BYTES,
```

here everywhere.